### PR TITLE
Fix npm run lint

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Detailed description**:
- Add `--fix` to lint to automatically fix some problems and include those fixes
- Fix `npm run lint` not working
- Fix `.prettierignore` not ignoring all non-JavaScript modules
- Remove duplicate `.prettierrc` since `prettier` block in `package.json` takes precedence

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

First commit contains my manual code changes. Second commit contains eslint automated fixes. Before adding `--fix`:
```
✖ 1413 problems (1133 errors, 280 warnings)
  344 errors and 0 warnings potentially fixable with the `--fix` option.
```

After adding `--fix`:
```
✖ 1053 problems (773 errors, 280 warnings)
```

**Checklist**
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

**Comment by steven-sheehy** [comment]
Draft since I will wait until #1714 to merge before pushing the automated lint fixes in the 2nd commit.

File: hedera-mirror-rest/monitoring/monitor_dashboard/js/main.js [hunk 1]
```diff
@@ -80,25 +80,14 @@ const makeTable = (data, server) => {
...
-      '</td>' +
-      '<td>' +
-      '<a = href="' +
-      result.url +
-      '">' +
-      result.message +
-      failureMsg +
-      '<a/></td>' +
-      '</tr>\n';
+      `${'<tr>' + '<tr>' + '<tr>' + '<td>' + '<span class="dot" style="background-color:'}${
```
Comment (ijungmann) [inline hedera-mirror-rest/monitoring/monitor_dashboard/js/main.js:83]: Is this first ${} needed?  Maybe I'm reading it wrong but it looks odd.

**Review by ijungmann — CHANGES_REQUESTED** [review]
Looks good, just the one question mentioned above.

File: hedera-mirror-rest/monitoring/monitor_dashboard/js/main.js (hunk 1 above)
Comment (steven-sheehy) [reply hedera-mirror-rest/monitoring/monitor_dashboard/js/main.js:83]: Probably not, this looks to be just eslint's best attempt at converting the previous string concats to templated string. I can tweak it further.

Comment (steven-sheehy) [reply hedera-mirror-rest/monitoring/monitor_dashboard/js/main.js]: Fixed along with some other minor improvements to that file.

**Review by Nana-EC — APPROVED** [review]
LGTM

File: hedera-mirror-rest/monitoring/monitor_dashboard/js/main.js (hunk 1 above)
Comment (ijungmann) [reply hedera-mirror-rest/monitoring/monitor_dashboard/js/main.js]: Much better

**Review by ijungmann — APPROVED** [review]
LGTM

**Pull request merged**